modules/consensus/synchronize.go

```diff
@@ -237,6 +242,11 @@ func (cs *ConsensusSet) threadedReceiveBlocks(conn modules.PeerConn) (returnErr
 // that BlockHeight onwards are returned. It also sends a boolean indicating
 // whether more blocks are available.
 func (cs *ConsensusSet) rpcSendBlocks(conn modules.PeerConn) error {
+	if err := cs.tg.Add(); err != nil {
+		return err
+	}
+	defer cs.tg.Done()
```

Comment:
should either be `threadedRPCSendBlocks` or should not have the `tg` code. Same applies to the other RPCs.